.github/workflows/master-e2e.yaml

```diff
@@ -417,6 +421,7 @@ jobs:
         if: inputs.test_type == 'cli'
         env:
           EMULATE_TPM: true
+          ISO_TO_TEST: ${{ inputs.iso_to_test }}
```

Comment:
It doesn't seem to be used in your code, maybe better to remove it. And anyway this may break the daily CI.